test/extern/fnPtrs/passBadFnsToCChapel-ChapelFcfToC.good

```diff
@@ -0,0 +1,2 @@
+passBadFnsToCChapel.chpl:18: error: unresolved call 'bar(chpl__fcf_type_void_void)'
+passBadFnsToCChapel.chpl:9: note: candidates are: bar(f: c_fn_ptr)
```

Comment:
Would it be possible to have this error message suggest that you should call c_fnPtrTo() on the function?